client/packages/inventory/src/Stocktake/DetailView/modal/StocktakeLineEdit/hooks/useDraftStocktakeLines.ts

```diff
@@ -13,6 +13,7 @@ export const useDraftStocktakeLines = (
   const [draftLines, setDraftLines] = useState<DraftStocktakeLine[]>([]);
 
   useEffect(() => {
+    if (!item) setDraftLines([]);
```

Comment:
Not directly related, but we weren't clearing the stocktake draft lines on ok+next, so the buttons were still enabled (which would submit the previous item again, causing an error)